NEWS.md

```diff
@@ -20,4 +22,341 @@
 
 # openxlsx 4.1.3
 
+## New Features
+
 * Added a `NEWS.md` file to track changes to the package.
+* Added `pkgdown` to create site.
+
+## Bug Fixes
+
+* Return values for cpp changed to R_NilValue for r-devel tests
+
+* Added empty lines at the end of files
+
+# openxlsx 4.1.2
+
+* Changed maintainer
+
+# openxlsx 4.1.1
+
+## New Features
+
+* `sep.names` allows choose other separator than '.' for variable names with a blank inside
+
+* Improve handling of non-region names in `getNamedRegions` and add related test
+
+# openxlsx 4.1.0
+
+## New Features
+
+* `deleteNamedRegions` to delete named region and optionally the worksheet data
+
+* set Workbook properties 'title', 'subject', 'category' 
+
+## Bug Fixes
+
+* `pageSetup` fails when passing in sheet by name
+
+* matching sheet names with special characters now works 
+
+* `skipEmptyCols` being ignored by `read.xlsx.Workbook`
+
+* zero column data.frames would throw an error.
+
+* `read.xlsx` on files created using apache poi failed to match sheet name to xml file. 
+
+* deleted table re-appearing after save & load.
+
+* newline characters in table names would corrupt file
+
+* datetime precision
+
+# openxlsx 4.0.17
+
+## New Features
+
+* `getNamedRegions` returns sheet name and cell references along with the named regions.
+
+* `borderStyle` and `borderColour` can be vector to specify different values for each side
+
+* `dataValidation` type "list"
+
+* `dataBar showValue`, gradient and border can now be set through conditionalFormatting()
+
+* options("openxlsx.zipflags") to pass additional flags to zip application e.g. compression level
+
+* `getTables()` and `removeTable()` to show and remove Excel table objects 
+
+* set column to 'hidden' with `setColWidths()`
+
+## Bug Fixes
+
+* `skipEmptyRows` & `skipEmptyCols` was being ignored by read.xlsx
+
+* date detection basic_string error
+
+* multiple spaces in table column names were not being maintained thus corrupting the xlsx file.
+
+* openXL fail silently on relative paths
+
+* `headerStyle` failed when writing a list of length 1 using write.xlsx
+
+* `detectDate` for read.xlsx issues
```

Comment:
* `detectDate` for `read.xlsx` issues

Could you also correct this line?

Reply:
If you use 2 white space between the asterix and the text the Codacy/PR Quality Review check will be successful.